blazor/diagram/scroll-settings.md

```diff
@@ -101,7 +101,92 @@ The Diagram control provides the following event for the scroll settings.
 
 }
 ```
+## AutoScroll 
+The auto-scroll feature automatically scrolls the diagram whenever the node is moved beyond its boundaries. Auto-scroll is enabled during node dragging and  node resizing, and multiple-selection operations.
 
+The diagram event [OnAutoScrollChange](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.SfDiagramComponent.html#Syncfusion_Blazor_Diagram_SfDiagramComponent_OnAutoScrollChange) gets triggered when the autoscroll (scrollbars) is changed, and you can do your own customization in this event. 
+
+The auto-scrolling region is limited by the [ScrollLimit](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.ScrollSettings.html#Syncfusion_Blazor_Diagram_ScrollSettings_ScrollLimit) property of the [ScrollSettings](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.ScrollSettings.html) class.
+
+The autoscroll behavior in your diagram can be enabled or disabled by using the [EnableAutoScroll](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.ScrollSettings.html#Syncfusion_Blazor_Diagram_ScrollSettings_EnableAutoScroll) property of the diagram. The following code example illustrates how to enabled/disabled the autoscroll support for node.
```

Comment:
The auto-scroll behavior in your diagram can be enabled or disabled by using the [EnableAutoScroll](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.ScrollSettings.html#Syncfusion_Blazor_Diagram_ScrollSettings_EnableAutoScroll) property of the diagram. The following code example illustrates enabling or disabling the auto-scroll support for nodes.